### PR TITLE
add global name

### LINKED ISSUE #1379: feature: add global name
APISIX need add global name for log platform, zipkin, such as spring boot 'spring.application.name',

### PR DESCRIPTION
**Pull request by qiujiayu**
Fix #1379

### CODE REVIEW

File: conf/config.yaml [hunk 1]
```diff
@@ -15,6 +15,7 @@
 # limitations under the License.
 #
 apisix:
+  name: APISIX                   # application name
```
Comment (membphis) [inline conf/config.yaml]: how about `APISIX gateway name`?

**Comment by membphis** [comment]
need test case

File: conf/config.yaml [hunk 2]
```diff
@@ -15,6 +15,7 @@
 # limitations under the License.
 #
 apisix:
+  name: APISIX                   # APISIX gateway name
```
Comment (moonming) [inline conf/config.yaml:18]: as discuss in mailing list:https://lists.apache.org/thread.html/r2856157a47272100b97cf8c93e93933edfcfc9724477625e531ee1b0%40%3Cdev.apisix.apache.org%3E, we'd better not add name in config.yaml.

Comment (membphis) [reply conf/config.yaml:18]: close it first

**Pull request closed without merging**